apps/docs/src/about/team.md

```diff
@@ -1,72 +1,128 @@
-    ]
+    core: true,
+    title: 'Engineer',
   },
   {
-    avatar: 'https://www.github.com/larsrickert.png',
+    login: "larsrickert",
     name: 'Lars Rickert',
-    title: 'Developer',
-    links: [
-      { icon: 'github', link: 'https://github.com/larsrickert' },
-    ]
+    core: true,
+    title: 'Engineer',
   },
   {
-    avatar: 'https://www.github.com/MajaZarkova.png',
+    login: "MajaZarkova",
     name: 'Maja Zarkova',
-    title: 'Developer',
-    links: [
-      { icon: 'github', link: 'https://github.com/MajaZarkova' },
-    ]
+    core: true,
+    title: 'Engineer',
   },
     {
-    avatar: 'https://www.github.com/ChristianBusshoff.png',
+    login: "ChristianBusshoff",
     name: 'Christian Bußhoff',
-    title: 'Developer',
-    links: [
-      { icon: 'github', link: 'https://github.com/ChristianBusshoff' },
-    ]
+    core: true,
+    title: 'Engineer',
   },
-]
-</script>
+  {
+    login: "rhoggs-bot-test-account",
+    type: "Bot",
+  },
+  {
+    login: "oemueller",
+    name: "Oliver Müller"
+  },
+  {
+    login: "markbrockhoff",
+    name: "Mark Brockhoff"
+  }
+];
 
-# Meet the team 👋
+const mapped = data.contributors.map((c) => ({
+  ...c,
+  avatar: c.avatar_url,
+  name: c.login,
+  links: [
+    { icon: 'github', link: c.html_url },
+  ],
+  ...overrides.find(n => c.login === n.login)
+}));
 
-onyx is maintained by [Schwarz IT](https://it.schwarz). Below you will find the members of our Core Team.
+const coreMembers = mapped.filter(m => m.core);
+const bots = mapped.filter(m => m.type === "Bot");
+const contributors = mapped.filter(m => !bots.includes(m) && !coreMembers.includes(m));
+</script>
 
-Are you looking for a bug report or feature request? Then please use our [GitHub issues](https://github.com/SchwarzIT/onyx/issues).
-For general Q&A, announcements and polls feel free to visit our community space via [GitHub discussions](https://github.com/SchwarzIT/onyx/discussions/categories/q-a).
+<VPTeamPage>
+  <VPTeamPageTitle>
+    <template #title>Meet the team 👋</template>
+    <template #lead>
+      onyx is maintained by a dedicated team at <a href="https://it.schwarz">Schwarz IT</a>. Below you will find the core members of our team.
+      <br><br>
+      Are you looking for a bug report or feature request?
+      <br><br>
+      Then please use our <a href="https://github.com/SchwarzIT/onyx/issues">GitHub issues</a>.
+      For general Q&A, announcements and polls feel free to visit our community space via <a href="https://github.com/SchwarzIT/onyx/discussions/categories/q-a">GitHub discussions</a>.
+    </template>
 
-<VPTeamMembers size="small" :members="members" />
+  </VPTeamPageTitle>
+  
+  <VPTeamMembers size="medium" :members="shuffleArray(coreMembers)" />
+  <VPTeamPageSection>
+    <template #title>Thank you to all contributors 🙏</template>
+    <template #members>
+      <VPTeamMembers size="small" :members="shuffleArray(contributors)" />
```

Comment:
I am not sure how this will scale in the future if we might have > 50 contributors since the page will get very large then...